js/binance.js

```diff
@@ -691,6 +691,13 @@ module.exports = class binance extends Exchange {
         return this.parseOrder (response);
     }
 
+    async createTimeInForceOrder (symbol, type, side, amount, price = undefined, timeInForce = undefined, params = {}) {
+        if (timeInForce === 'ioc')
```

Comment:
Well, if someone calls it with `timeInForce = 'foc'` (or `timeInForce = 'xxx'` it will get placed while it should not.